server/map_rpc_server.go

```diff
@@ -156,6 +156,34 @@ func (t *TrillianMapServer) GetLeavesByRevision(ctx context.Context, req *trilli
 	return t.getLeavesByRevision(ctx, req.MapId, req.Index, req.Revision)
 }
 
+// GetLeavesByRevisionNoProof implements the GetLeavesByRevision RPC method.
```

Comment:
Can we add tests for the new RPC?

Reply:
Tests added

Reply:
There don't seem to be RPC server unit tests though.